npm/oxlint/bin/oxlint

```diff
@@ -20,7 +20,8 @@ const isMuslFromFilesystem = () => {
   const { readFileSync } = require("fs");
   try {
     return readFileSync("/usr/bin/ldd", "utf-8").includes("musl");
-  } catch {
+  } catch(error) {
+    console.error(error);
```

Comment:
The catch is used for catching "file not found", so the console.log is unncessary.

Reply:
Done!